lib/discourse_slack/api.rb

```diff
@@ -0,0 +1,72 @@
+module DiscourseSlack
+  class API
+
+    BASE_URL = "https://slack.com/api/"
+
+    def self.http
+      @http ||= begin
+        http = Net::HTTP.new("slack.com" , 443)
+        http.use_ssl = true
+
+        http
+      end
+    end
+
+    def self.get_request(uri)
+      response = http.request(Net::HTTP::Get.new(uri))
+
+      return JSON(response.body) if response && response.code == "200"
+
+      { "error": I18n.t('slack.errors.invalid_response') }
```

Comment:
This error is too generic. We should just send the response body to the client or raise an error. I had a hard time figuring out why I wasn't getting the feature to work.